src/io.c

```diff
@@ -85,3 +83,14 @@ chomp(char *string, ssize_t length)
 		string[length - 1] = '\0';
 	}
 }
+
+static char *
+eager_strpbrk(const char *string, const char *separators) {
+	char *ptr = NULL, *tmp_ptr;
+	for (tmp_ptr = strpbrk(string, separators);
+	     tmp_ptr;
+	     tmp_ptr = strpbrk(tmp_ptr, separators)) {
```

Comment:
Is this the same as the following and would you in that case think the following is clearer?

```c
tmp_ptr = strpbrk(string, separators);
while (tmp_ptr = strpbrk(tmp_ptr, separators)) {
```